.eslintrc

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Is this file a result of your upgrading Angular?

I notice you haven't also upgraded Nx. Do you know how to do this?

(Upgrading Angular without also upgrading Nx can be a problem)

Reply:
upgraded Nx